### ISSUE TITLE
assembly browser: cannot browse interface declaration which contains default interface methods

### ISSUE THREAD

**Issue by atsushieno** [mono/monodevelop#5348]
I'm attaching an example assembly that contains an interface with default interface methods (DIMs) and an implementing class. The DIM can be disassembled and shown, so as the implementing class declaration, but the interface declaration is somehow not displayed.

![screenshot from 2018-07-10 18-46-42](https://user-images.githubusercontent.com/53929/42502663-a763c1c6-8471-11e8-9a6c-e06ee03d9c67.png)
![screenshot from 2018-07-10 18-46-58](https://user-images.githubusercontent.com/53929/42502675-b01084f8-8471-11e8-8f36-3e8fcaf9b95d.png)

[Xamarin.Android.DefaultInterfaceMethods-Lib.dll.zip](https://github.com/mono/monodevelop/files/2179808/Xamarin.Android.DefaultInterfaceMethods-Lib.dll.zip)

It might be either cecil, or assembly browser addin, or anything inbetween, but so far I'm recording it here. (I'm asking jbevain too, will update if I get any hints.)

**Comment by atsushieno** [comment]
(Maybe obvious but forgot to note: it makes use of default interface methods which is available only in C# 8.0.)

**Comment by jbevain** [comment]
I don't think this is a Cecil bug, I can definitely browse the method bodies in the interfaces.

**Comment by mkrueger** [comment]
Fixed in newer versions.

![Bildschirmfoto 2020-06-05 um 09 20 27](https://user-images.githubusercontent.com/341098/83848330-00061200-a70e-11ea-9226-aac65d68ab2d.png)
